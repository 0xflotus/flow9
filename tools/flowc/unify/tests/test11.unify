import piece;
//α29 ={PieceFieldHide<> .. <union> ::= (PieceFieldHide<>, PieceFieldIsSlider<>, PieceFieldIuEditor<>)};
//α26 ={PieceFieldTooltip<> .. <union> ::= (PieceFieldHide<>, PieceFieldIsSlider<>, PieceFieldIuEditor<>, PieceFieldTooltip<>)};
// α29 grow_right  α26;
//[α29 ]  grow_right  [α26];
//[<union> ::= (PieceFieldHide<>, PieceFieldIsSlider<>, PieceFieldIuEditor<>)]  reduce_left  [α26]


α20 ={<union> ::= (PieceFieldHide<>, PieceFieldIsSlider<>, PieceFieldIuEditor<>, PieceFieldTooltip<>) .. *};
α18 =[α29];
[α20]  grow_right  α18;
