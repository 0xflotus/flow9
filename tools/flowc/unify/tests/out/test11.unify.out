α18=[{<union> ::= (PieceFieldHide<>, PieceFieldIsSlider<>, PieceFieldIuEditor<>, PieceFieldTooltip<>) .. *}]
α20={<union> ::= (PieceFieldHide<>, PieceFieldIsSlider<>, PieceFieldIuEditor<>, PieceFieldTooltip<>) .. *}
α29={<union> ::= (PieceFieldHide<>, PieceFieldIsSlider<>, PieceFieldIuEditor<>, PieceFieldTooltip<>) .. *}
